client/jetpack-cloud/sections/partner-portal/primary/payment-method-add/index.tsx

```diff
@@ -25,11 +180,107 @@ export default function PaymentMethodAdd(): ReactElement {
 				<CardHeading size={ 36 }>{ translate( 'Payment Method' ) }</CardHeading>
 			</div>
 
-			<Card className="payment-method-add__body">
-				<div className="payment-method-add__body-left"></div>
+			<CheckoutProvider
+				onPaymentComplete={ () =>
+					onPaymentSelectComplete( {
+						successCallback: () => page( '/partner-portal/payment-method/' ),
+						translate,
+						showSuccessMessage,
+						purchase,
+					} )
+				}
+				showErrorMessage={ showErrorMessage }
+				showInfoMessage={ showInfoMessage }
+				showSuccessMessage={ showSuccessMessage }
+				paymentMethods={ paymentMethods }
+				paymentProcessors={ {
+					card: ( data ) =>
+						assignNewCardProcessor(
+							{
+								purchase,
+								useAsPrimaryPaymentMethod,
+								translate,
+								stripe,
+								stripeConfiguration,
+								reduxDispatch: dispatch,
+							},
+							data
+						),
+				} }
+				isLoading={ isStripeLoading }
+				initiallySelectedPaymentMethodId="card"
+			>
+				<Card className="payment-method-add__content">
+					<CardHeading>{ translate( 'Credit card details' ) }</CardHeading>
+
+					{ 0 === paymentMethods.length && <CreditCardLoading /> }
 
-				<div className="payment-method-add__body-right"></div>
-			</Card>
+					{ currentPaymentMethodNotAvailable && purchase && (
+						<CurrentPaymentMethodNotAvailableNotice purchase={ purchase } />
+					) }
+
+					{ paymentMethods && paymentMethods[ 0 ] && paymentMethods[ 0 ].activeContent }
+
+					{ ! purchase && (
+						<FormLabel className="payment-method-add__all-subscriptions-checkbox-label">
+							<FormInputCheckbox
+								className="payment-method-add__all-subscriptions-checkbox"
+								checked={ useAsPrimaryPaymentMethod }
+								onChange={ () => setUseAsPrimaryPaymentMethod( ( checked ) => ! checked ) }
+								aria-label={ assignAllSubscriptionsText }
+							/>
+							<span>{ assignAllSubscriptionsText }</span>
+						</FormLabel>
+					) }
+
+					<div className="payment-method-add__navigation-buttons">
+						<Button
+							className="payment-method-add__back-button"
+							href="/partner-portal/payment-method/"
+							disabled={ isStripeLoading }
+							onClick={ onGoToPaymentMethods }
+						>
+							{ translate( 'Go back' ) }
+						</Button>
+						<CheckoutSubmitButton
+							className="payment-method-add__submit-button"
+							disabled={ isEditing }
+						/>
+					</div>
+					{ isEditing && (
+						<Button
+							className="payment-method-add__delete-button"
+							scary
+							borderless
+							onClick={ openDeleteDialog }
+						>
+							{ translate( 'Delete Payment Method' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **16** times:
`translate( 'Delete payment method' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/partner-portal-delete-payment-method)